core/include/cib.h

```diff
@@ -39,6 +39,9 @@ typedef struct {
 
 /**
  * @brief   Initialize cib_t to a given size.
+ *
+ * @param[in]  SIZE     Size of the buffer, must not exceed `UINT_MAX` + 1.
```

Comment:
values here and below don't match?

Reply:
Yes, because in the function below the value is restricted by the value space of the parameter, here it is not, and it needs only be max(value_space) - 1 for the member. So yes, the values don't match but so does the behavior.

Reply:
But the size needs to be limited so reads and writes do not reach  `-1` when casted to int. The restriction for CIB_INIT is the same as for cib_init().

Reply:
Ah right. When `mask == UINT_MAX`, `cib_full()` can never be true!

Reply:
Adapted.